text/0148-sampling-probability.md

```diff
@@ -0,0 +1,376 @@
+# Probability sampling of telemetry events
+
+Specify a foundation for sampling techniques in OpenTelemetry.
+
+## Motivation
+
+In tracing, metrics, and logs, there are widely known techniques for
+sampling a stream of events that, when performed correctly, enable
+collecting a fraction of the complete data while maintaining
+substantial visibility into the whole population of events.
+
+These techniques are all forms of approximate counting.  Estimates
+calculated by the forms of sampling outlined here are considered
+accurate, in that they are random variables with an expected value
+equal to the true count.  With enough observations, 
```

Comment:
```suggestion
equal to the true count.
```

This seems to be dangling here, is it missing somewhere down below?

Reply:
Removed. Thanks.